server.go

```diff
@@ -403,7 +413,12 @@ func (s *Server) consume(binding HandlerBinding, inputCh *amqp.Channel, wg *sync
 	return consumerTag, nil
 }
 
-func (s *Server) runHandler(handler HandlerFunc, deliveries <-chan amqp.Delivery, queueName string, wg *sync.WaitGroup) {
+func (s *Server) runHandler(
```

Comment:
What kind of new style guide have you started following :D

Reply:
I've started trying this: https://github.com/segmentio/golines

Do you think we should not? I kind of like it.

Reply:
I have no strong opinions about this so go ahead! Just not used to it, but it's very clear and in a separate commit so I'm totally fine with this!